src/renderers/dom/client/ReactMount.js

```diff
@@ -315,10 +315,6 @@ var ReactMount = {
     shouldReuseMarkup,
     context
   ) {
-    if (__DEV__) {
```

Comment:
See last commit—I replaced most of these blocks with transactions. (I can revert just that commit if you’re not into this.)